docs/_sass/_landingPage.scss

```diff
@@ -0,0 +1,721 @@
+            color: #8f8f8f;
+        }
+    }
+}
+
+.backToTopHidden {
+    display: none;
+}
+
+.backToTopVisible {
+    width: 44px;
+    height: 44px;
+    line-height: 44px;
+    box-shadow: 0 1px 4px 0 #d3d3d3;
+    border: solid 1px #d3d3d3;
+    background-color: #ffffff;
+    overflow: hidden;
+    z-index: 999;
+    cursor: pointer;
+    position: fixed;
+    bottom: 100px;
+    right: 30px;
+    border-radius: 50%;
+    color: $home-page-action-color;
+    text-align: center;
+    font-size: 20px;
+    text-decoration: none;
+}
+
+
+// TABLET AND MOBILE
+
+@media (max-width: 1024px) { 
+    .header-section {
+        &__links a{
+            margin: 0 6px;  
+        }
+    }
+}
+
+@media (max-width: 768px) { 
+    .header-section {
+        padding: 0 20px;
+        justify-content: space-between;
+        &__links {
+            display: none;
+        }
+
+        &__logo {
+            flex: 1;
+            text-align: center;
+        }
+
+        &__hamburger {
+            display: block;
+        }
+    }
+}
+
+
+@media (max-width: 1224px){
+    .intro-section {
+        &__text {
+            width: 100%;
+        }
+    }
+}
+
+@media (max-width: 768px){
+    .intro-section {
+        padding: 36px;
+
+        &__text {
+           width: 100%;
+           @include item-title-mobile();
+        }
+
+        &__logo {
+            height: 70px;
+        }
+    }
+}
+
+@media (max-width: 900px){
+    .libraries-section {
+        padding: 80px 120px;
+        &__container {
+            grid-template-columns: auto;
+        }
+    }
+}
+@media (max-width: 768px){
+    .libraries-section {
+        padding: 64px 44px;
+
+        &__container {
+            grid-template-columns: auto;
+        }
+
+        &__title {
+            @include title-mobile();
+            margin-bottom: 44px;
+        }
+
+        &__item {
+            min-height: 260px;
+        }
+
+        &__item-title {
+            @include item-title-mobile();
+            margin-bottom: 8px;
+        }
+
+        &__text {
+            @include text-mobile();
+        }
+
+        &__more {
+            font-size: 14px;
+            line-height: 1.43;
+        }
+    }
+}
+
+@media (max-width: 600px){
+    .libraries-section {
+        padding: 64px 0;
+
+        &__container {
+            grid-gap: 0;
+        }
+
+        &__title {
+            @include title-mobile();
+            margin-bottom: 20px;
+        }
+
+        &__item {
+            min-height: auto;
+            border: none;
+            box-shadow: none;
+            border-radius: 0;
+            border-bottom: solid 1px #d3d3d3;
+            transition: none;
+
+            &:hover {
+                box-shadow: none;
+                transform: none;
+            }
+        }
+    }
+}
+
+@media (max-width: 1024px){
+    .features-section {
+        &__container {
+            grid-template-columns: auto auto;
+        }
+    }
+}
+
+@media (max-width: 768px){
```

Comment:
What about adding only one media query per `max-width`?
From largest (1024px) to smallest (600px)

Reply:
The rest is perfect.

Reply:
Doesn't look good without this breakpoint. O.w. it's not a problem to change it from 768 to 600px

Reply:
This is how the section looks like at 630px:
<img width="539" alt="Screen Shot 2020-01-07 at 8 17 48 AM" src="https://user-images.githubusercontent.com/39598672/71898081-60286600-3126-11ea-8fe0-d090b072d78e.png">

This is how the section looks like compared to the other sections:
<img width="504" alt="Screen Shot 2020-01-07 at 8 18 12 AM" src="https://user-images.githubusercontent.com/39598672/71898113-7afada80-3126-11ea-9c82-dced180e3a8c.png">
<img width="592" alt="Screen Shot 2020-01-07 at 8 18 32 AM" src="https://user-images.githubusercontent.com/39598672/71898117-7cc49e00-3126-11ea-8b6f-17defc74dd19.png">